### DISCUSSION TITLE
Active Directory integration (MFA enabled)

### DISCUSSION THREAD

**Discussion by johnununu** [Security-Onion-Solutions/securityonion#7595] [Unsupported Versions] [answered]
Now that MFA is enabled and functioning **THANK YOU!!!!!!**  is there thought into AD/LDAP integration

**Comment by dougburks** [comment, marked as answer]
From https://docs.securityonion.net/en/2.3/faq.html#can-i-connect-security-onion-to-active-directory-or-ldap:
> [](https://docs.securityonion.net/en/2.3/faq.html#can-i-connect-security-onion-to-active-directory-or-ldap)Can I connect Security Onion to Active Directory or LDAP?
We understand the appeal of integrating with directory services like Active Directory and LDAP, but we typically recommend against joining any security infrastructure (including Security Onion) to directory services. The reason is that when you get an adversary inside your network, one of their first goals is going to be gaining access to that directory. If they get access to the directory, then they get access to everything connected to the directory. For that reason, we recommend that all security infrastructure (including Security Onion) be totally separate from directory services.

**Reply by johnununu** [reply]
I read that.  Now that MFA is an option, I didn't know if Active Directory integration might be back on the table.
Should a threat actor gain access to the domain, they would still need access to that accounts device.
